test/test_dimension.py

```diff
@@ -79,3 +86,35 @@ def test_arithmetic_ops_preserve_type_array(
 
     c = a / 11
     assert all(c == u * [2 / 11, 3 / 11])
+
+
+@pytest.mark.parametrize('dimension', _ALL_DIMENSIONS)
+def test_arithmetic_ops_preserve_type_multi_array(
```

Comment:
from TypedUnits perspective there is no difference between a 1-D array and a multidimensional, both are a numpy.ndarray so this case is implicitly covered by the tests.

We can consider explicitly testing for this though but for that please improve the test.